### ISSUE TITLE
[Feat]: I want to add my profile

### ISSUE THREAD

**Issue by Adithyasharma1162** [codeaashu/DevDisplay#379]
### What feature?

I want to add my profile

### Add screenshots

Add screenshots

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Adithyasharma1162** [comment]
Can u label it with hacktoberfest, opensource and assign it to me

**Comment by ishap11** [comment]
Hi @Adithyasharma1162  , thanks for your interest in contributing to DevDisplay!

Before we proceed, we noticed you added your profile on #DevDisplay but forgot to star the repo. It's like making an amazing sandwich and forgetting to take a bite! -- I think our repo's feeling a little lonely without your 🤗 So, be sure to star the DevDisplay GitHub Repository. 🌟

💌 This is a mandatory step for all contributors. & It's is the first priority of contribution in DevDisplay 💌

🚀 We look forward to your contribution once these steps are completed. 🚀

Thanks 👍🏻
